integration-cli/docker_cli_build_test.go

```diff
@@ -2071,7 +2071,7 @@ func (s *DockerSuite) TestBuildContextCleanup(c *check.C) {
 	testRequires(c, SameHostDaemon)
 
 	name := "testbuildcontextcleanup"
-	entries, err := ioutil.ReadDir(filepath.Join(dockerBasePath, "tmp"))
+	entries, err := ioutil.ReadDir(filepath.Join(testEnv.DockerBasePath(), "tmp"))
```

Comment:
Possible future enhancement:

```go
testEnv.DockerPath("tmp")
```

It seems this is used with `filepath.Join()` in (almost) every case, so it could be moved into the helper.